vms/platformvm/camino_service.go

```diff
@@ -74,6 +75,8 @@ func (response GetBalanceResponseWrapper) MarshalJSON() ([]byte, error) {
 
 // GetBalance gets the balance of an address
 func (s *CaminoService) GetBalance(_ *http.Request, args *GetBalanceRequest, response *GetBalanceResponseWrapper) error {
+	s.vm.ctx.Log.Debug("Platform: GetBalance called")
```

Comment:
Is this left over from debugging or it's needed?

Reply:
It is in every (avax and our) api call, if we arrive strange traffic in node we can see in debug log if something is called very often. So no, no debugging, it was simply missing....